### PR TITLE
Adding docs for prometheus.

### PR DESCRIPTION
**Pull request by shawn-hurley**
**Describe what this PR does and why we need it**:
Adding documentation for using the metrics endpoint

**NOTE: does the troubleshooting doc make sense?**
Changes proposed in this pull request
 - Add documentation on how to use the prometheus endpoint

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
...
+
+1. The user that you are logged in as will need to have access to `cluster-debugger-role`.
+2. The broker url will need to be exposed and reachable.
+
+#### Current ASB Metrics Exposed
+
+1. asb_sandbox - Keeps track of the active sandboxes
+2. asb_specs_loaded - will keep track of the number of specs currently loaded, will be reset on bootstrap. Labels can be used to determine the registry that has X number loaded.
+3. asb_spec_reset - will keep track of how many times the specs have been reset.
+4. asb_provision_jobs - will keep track of how many jobs are currently in the provision buffer?
```
Comment (eriknelson) [inline docs/troubleshooting.md]: Why the question mark?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
 $ oc rollout latest asb
 deploymentconfig "asb" rolled out
 ```
+
+### Metrics
```
Comment (djzager) [inline docs/troubleshooting.md]: :+1: Excellent place for this!

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
...
+
+The broker will expose [Prometheus](https://prometheus.io/) style metrics that you can use to troubleshoot the broker if none of the steps above work. You can access the published metrics by calling:
```
Comment (djzager) [inline docs/troubleshooting.md]: Optional: 
The broker exposes [Prometheus](https://prometheus.io/) style metrics for monitoring and troubleshooting purposes. You can access these metrics via the `/metrics` endpoint by calling:

Reason: It's infinitely useful by itself (as demonstrated already) and isn't conditional on the rest of the troubleshooting steps.

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
...
+2. The broker url will need to be exposed and reachable.
+
+#### Current ASB Metrics Exposed
+
+1. asb_sandbox - Keeps track of the active sandboxes
+2. asb_specs_loaded - will keep track of the number of specs currently loaded, will be reset on bootstrap. Labels can be used to determine the registry that has X number loaded.
+3. asb_spec_reset - will keep track of how many times the specs have been reset.
+4. asb_provision_jobs - will keep track of how many jobs are currently in the provision buffer
+5. asb_deprovision_jobs - will keep track of how many jobs are currently in the deprovision buffer
+6. asb_update_jobs - will keep track of how many jobs are currently in the update buffer?
```
Comment (rthallisey) [inline docs/troubleshooting.md]: ```buffer?``` Drop off the question mark and add periods to all these.

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -188,3 +188,27 @@ Redeploy the broker using origin clients `rollout latest` command.
...
+5. asb_actions_requested - keeps track of the number of actions requested correctly (broken down by action = bind,unbind,update,provision,deprovision)
```
Comment (rthallisey) [inline docs/troubleshooting.md]: ```requested correctly``` I think I know what you mean by this, but can you say this another way?

I view this metric as: The total number of successful bind, unbind, update, provision, and deprovision requests. Each are labeled and can be filtered in a dashboard.

**Pull request merged**